obm-autoset/obm_config.json

```diff
@@ -0,0 +1,23 @@
+{ "ora": {

```

Comment:
meanwhile, this is just an example file, it's better to add the word "example" in the file name. You can refer to this project https://github.com/etsy/hound#using-go-tools to see how it specify the config file.